src/components/comment-list-item.component.js

```diff
@@ -143,13 +143,15 @@ class CommentListItemComponent extends Component {
       translate('issue.comment.deleteAction', language),
     ];
 
-    const isActionButtonVisible =
+    const isActionMenuEnabled =
       comment.user &&
       authUser.login === comment.user.login &&
       !this.isIssueDescription();
 
     return (
-      <TouchableWithoutFeedback onLongPress={this.showMenu}>
+      <TouchableWithoutFeedback
+        onLongPress={() => isActionMenuEnabled && this.showMenu()}
+      >
```

Comment:
How about the `disabled` attribute? It can avoid creating an anonymous function in `render`.